setup.py

```diff
@@ -56,15 +56,11 @@
 
         # Specify the Python versions you support here. In particular, ensure
         # that you indicate whether you support Python 2, Python 3 or both.
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
     ],
 
     # Require Python version equal or more than Python 3.
```

Comment:
Not your change, but should be `equal or higher than` and also adjust the version requirement in the comment.